utils/utils_test.go

```diff
@@ -174,6 +174,7 @@ func TestIsWildcardParentheses(t *testing.T) {
 }
 
 func TestAntPathToRegExp(t *testing.T) {
```

Comment:
To make this test function more readable, let's decribe the roles of the various components, and how they relate to each other to test the functionality - 
* fileSystemPaths
* The properties for the ```tests``` struct.